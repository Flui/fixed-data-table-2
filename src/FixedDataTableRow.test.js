import React from 'react';
import FixedDataTableRow from 'FixedDataTableRow';
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
  const props = {
    width: 100,
    height: 100,
    offsetTop: 0,
    rowsCount: 0,
    rowHeight: 20,
    fixedColumns: [],
    index: 0,
    scrollableColumns: 0,
    scrollLeft: 10,
    scrollTop: 20,
  }

  const renderer = new ShallowRenderer();
  renderer.render(<FixedDataTableRow {...props}></FixedDataTableRow>);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});