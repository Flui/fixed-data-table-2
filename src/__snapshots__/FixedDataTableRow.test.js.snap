// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixedDataTableRowLayout_rowWrapper"
  style={
    Object {
      "height": 100,
      "left": "0px",
      "top": "0px",
      "width": 100,
      "zIndex": 0,
    }
  }
>
  <FixedDataTableRowImpl
    fixedColumns={Array []}
    height={100}
    index={0}
    offsetTop={undefined}
    rowHeight={20}
    rowsCount={0}
    scrollLeft={10}
    scrollTop={20}
    scrollableColumns={0}
    width={100}
    zIndex={undefined}
  />
</div>
`;
